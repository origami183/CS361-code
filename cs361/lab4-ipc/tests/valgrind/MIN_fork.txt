==1844124== Memcheck, a memory error detector
==1844124== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1844124== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1844124== Command: ../ipc ../data/f1.txt
==1844124== 
CKSUM(../data/f1.txt) = '3015617425'
==1844124== 
==1844124== HEAP SUMMARY:
==1844124==     in use at exit: 0 bytes in 0 blocks
==1844124==   total heap usage: 3 allocs, 3 frees, 9,021 bytes allocated
==1844124== 
==1844124== All heap blocks were freed -- no leaks are possible
==1844124== 
==1844124== For lists of detected and suppressed errors, rerun with: -s
==1844124== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
