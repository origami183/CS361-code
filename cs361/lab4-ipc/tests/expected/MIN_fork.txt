CKSUM(../data/f1.txt) = '3015617425'
